test/one/profiler/test/Runner.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package one.profiler.test;
+
+import java.io.File;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.regex.Pattern;
+
+public class Runner {
+    private static final Logger log = Logger.getLogger(Runner.class.getName());
+
+    private static final Os currentOs = detectOs();
+    private static String libExt;
```

Comment:
Can be also made `final`. Alternatively, the field could be removed altogether, as the value could be deduced from `currentOs`.

Reply:
Addressed